src/lib/git.js

```diff
@@ -166,8 +171,14 @@ async function sha(dir, branch) {
   return await git(dir, "show-ref", "-s", `refs/remotes/origin/${branch}`);
 }
 
-async function rebase(dir, branch) {
-  return await git(dir, "rebase", "--quiet", "--autosquash", branch);
+async function rebase(dir, remote, branch) {
```

Comment:
the same here, wdyt?
```suggestion
async function rebase(dir, branch, remote = "upstream") {
```

Reply:
Added!